code/modules/food&drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -162,6 +162,7 @@
 				icon_state = "tequilasunriseglass"
 				name = "tequila Sunrise"
 				desc = "Oh great, now you feel nostalgic about sunrises back on Terra..."
+			// Toxins Special is also in drinks.dm because shot glasses, mirror any changes there. //
```

Comment:
it's less specifically the drinks themselves and the fact that you're having to ask for future changes to be mirrored, hint, nobody is going to do so and if they do then they are a really nice person.

These drinks appear with shotglass icons, now we have actual shotglasses you should just remove the ability to mix said drinks in normal glasses.